docs/articles/user_acquisition_analytics.md

```diff
@@ -0,0 +1,448 @@
+    InMemoryExecutor,
+    InMemoryApp,
+)
+from superlinked.framework.dsl.index.index import Index
+from superlinked.framework.dsl.source.in_memory_source import InMemorySource
+from superlinked.framework.dsl.space.text_similarity_space import TextSimilaritySpace
+from superlinked.framework.dsl.space.number_space import NumberSpace
+from superlinked.framework.dsl.space.recency_space import RecencySpace
+
+alt.renderers.enable("mimetype")  # important: to render plots in colab, replace 'mimetype' with 'colab'
+alt.data_transformers.disable_max_rows()
+os.environ["TOKENIZERS_PARALLELISM"] = "false"
+pd.set_option("display.max_colwidth", 190)
+pd.options.display.float_format = "{:.2f}".format
+```
+
+Now we import our dataset. 
+
+```python
+dataset_repository_url = (
+    "https://storage.googleapis.com/superlinked-notebook-user-acquisiton-analytics"
+)
+USER_DATASET_URL = f"{dataset_repository_url}/user_acquisiton_data.csv"
+NOW_TS = 1708529056
+EXECUTOR_DATA = {CONTEXT_COMMON: {CONTEXT_COMMON_NOW: NOW_TS}}
+```
+
+(This might take a few minutes. While you're waiting, you can always find other interesting reading in [VectorHub](https://superlinked.com/vectorhub/).)
+
+## Read and explore our dataset
+
+Now that our dataset's imported, let's take a closer look at it:
+
+```python
+NROWS = int(os.getenv("NOTEBOOK_TEST_ROW_LIMIT", str(sys.maxsize)))
+user_df: pd.DataFrame = pd.read_csv(USER_DATASET_URL, nrows=NROWS)
+print(f"User data dimensions: {user_df.shape}")
+user_df.head()
+```
+
+We have 8000 users, and (as we can see from the first five rows) 4 columns of data:
+
+![first 5 rows of our dataset](../assets/use_cases/user_acquisition_analytics/user_id-sign_up_date-ad_creative-activity.png)
+
+To understand which ad creatives generated how many signups, we create a DataFrame:
+
+```python
+pd.DataFrame(user_df["ad_creative"].value_counts())
+```
+
+...which looks like this:
+
+![ad_creatives by count](../assets/use_cases/user_acquisition_analytics/ad_creative-count.png)
+
+The influencer (XYZCr$$d) backed ad creatives seem to have worked better - generating many more signups (total 5145) than the August campaign ad creatives (total 2855). 
+
+Now, let's take a look at the **distribution of users by activity level** (api calls/day).
+
+```python
+alt.Chart(user_df).mark_bar().encode(
+    alt.X("activity:Q", bin=True, title="Activity count"),
+    alt.Y("count()", title="# of users"),
+).properties(width=600, height=400)
+```
+
+![distribution by activity count](../assets/use_cases/user_acquisition_analytics/user_distrib-activity_count.png)
+
+The activity (api calls/day) distribution is bimodal. Here, it's possible that the first activity level group (0.00-0.40 api calls/day) may be largely new users, who - if they signed up in the most recent (December) campaign - have less time to accumulate activity than users who signed up earlier. (Note: we can use derive our NumberSpace min and max from this distribution.)
```

Comment:
I used api calls/day to eliminate this effect. I'd rather say that most people have 0 to very low activity (signed up, maybe did something and never returned, or occasionally return for repeated campaigns), and for active users activity has a distribution of which the mode is around .6, .8 api call/day (means returning every some  (2-3) day trigerring some (1-2) api calls

Reply:
I was going off your comments to the code for generating this distribution:
"
# The distribution of activity shows a 2-mode distribution
# possibly in connection with the two campaigns for user acquisition as newer users
# did not have time to accumulate activity like older users.
# Also this informs NumberSpace parameters
"

Reply:
Right, then I have a disagreement with myself :)

fixed the notebook now with a very similar text I wrote here